projects/cloud_feedbacks/index.md

```diff
@@ -1,463 +1,834 @@
----
-layout: cloud_feedbacks
-title: Cloud Feedbacks
----
+--- layout: cloud_feedbacks title: Cloud Feedbacks ---
 
 ## Identifying Robust Cloud Feedbacks in Observations and Models
-
-
-![Average longwave (LW), shortwave (SW), and net cloud feedbacks as simulated by global climate models]({{site.baseurl}}/projects/cloud_feedbacks/cloud_feedback_200h.png)
+## 
```

Comment:
The `##` and `###` with no text proceeding them will show up in the HTML when Jekyll processes this file.  Do you want these?

Reply:
Thanks. I've fixed them in my latest commit.